### PR TITLE
Resolve PE-D issues - Reminder

### PR DESCRIPTION
**Pull request by FeliciaTay**

### CODE REVIEW

File: src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.status.Status;
+import seedu.address.testutil.InternshipApplicationBuilder;
+
+class StatusIsWishlistPredicateTest {
+    @Test
+    public void equals() {
+        StatusIsInterviewPredicate predicate = new StatusIsInterviewPredicate();
```
Comment (ongch00) [inline src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java]: Should be StatusIsWishlistPredicate

File: src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    @Test
+    public void test_statusIsInterview_returnsTrue() {
+        StatusIsWishlistPredicate predicate = new StatusIsWishlistPredicate();
+
+        assertTrue(predicate.test(new InternshipApplicationBuilder().withStatus(Status.WISHLIST).build()));
+    }
+
+    @Test
+    public void test_statusIsNotInterview_returnsFalse() {
```
Comment (ongch00) [inline src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java]: I think interview needs to be changed to wishlist for method names

**Review by ongch00 — COMMENTED** [review]
Other than that one naming issue in the StatusIsWishlistPredicateTest, I dont see any obvious issues

File: src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java (hunk 1 above)
Comment (FeliciaTay) [reply src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java]: Oops, thanks for catching this!

File: src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java (hunk 2 above)
Comment (FeliciaTay) [reply src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java]: This as well!

**Comment by ongch00** [comment]
Alright looks good, Thanks!

**Pull request merged**